starknet/src/utils/simple_majority.cairo

```diff
@@ -0,0 +1,179 @@
+use starknet::info;
+use sx::types::{Proposal, FinalizationStatus, ProposalStatus};
+
+/// Returns the status of a proposal, according to a 'Simple Majority' rule.
+/// 'Simple Majority' is defined like so: a proposal is accepted if there are more `votes_for` than `votes_against`.
+/// So, a proposal will return Accepted if `max_end_timestamp` has been reached, and `votes_for > votes_agasint`.
+/// A proposal will return `VotingPeriodAccepted` if `min_end_timestamp` has been reached and `votes_for > votes_against`.
+fn get_proposal_status(
+    proposal: @Proposal, votes_for: u256, votes_against: u256, votes_abstain: u256,
+) -> ProposalStatus {
+    let accepted = votes_for > votes_against;
+
+    let timestamp = info::get_block_timestamp().try_into().unwrap();
+    if *proposal.finalization_status == FinalizationStatus::Cancelled(()) {
+        ProposalStatus::Cancelled(())
+    } else if *proposal.finalization_status == FinalizationStatus::Executed(()) {
+        ProposalStatus::Executed(())
+    } else if timestamp < *proposal.start_timestamp {
+        ProposalStatus::VotingDelay(())
+    } else if timestamp < *proposal.min_end_timestamp {
+        ProposalStatus::VotingPeriod(())
+    } else if timestamp < *proposal.max_end_timestamp {
```

Comment:
i think just ignore `min_end_timestamp`

Reply:
Hmm so the rationale behind that was:
- those who don't want the early end can set min/max to the same value
- those who want to have an early end (for whatever reason) can still have it without having to write another strategy

Reply:
the thing is min/max are space params they cant be set just for proposals with this strategy. 

The goal of a 'simple majority' strategy is to find the majority of VP over the voting period. i dont think it would ever make sense to have an early end. So if spaces are using this execution strategy in combination with others, they would need to update space params often which would be annoying.

Reply:
fair enough ! f6ac7e04a8b18dc21688712bcd7acb7a1a1a2dcb